codeql_ruby/cwe-089/PgInjection.rb

```
@@ -0,0 +1,99 @@
+
+    # .prepare() and .exec_prepared()
+    # NOT EXECUTED: SQL statement constructed from user input but not executed
+    qry3 = "SELECT * FROM users WHERE username = '#{name}';"
+    conn.prepare("query_3", qry3)
+  end
+# {/fact}
+end
+
+class BarController < ApplicationController
+  def safe_paths
+    name1 = params["name1"]
```

RuboCop (1.80.2)

[warning] Useless assignment to variable - `name1`. Did you mean `name`?

(Lint/UselessAssignment)